mlflow/keras.py

```diff
@@ -123,11 +123,13 @@ def __init__(self, keras_model, graph, sess):
         self._graph = graph
         self._sess = sess
 
-    def predict(self, dataframe):
+    def predict(self, input_data):
```

Comment:
Shouldn't `input_data` always be a data frame because this is the standard input format for pyfunc-flavored models?

Reply:
it's not enforced so it could be e.g. numpy array.

there is an isinstance test on line 130

Reply:
Our documentation indicates that the input format should be `pandas.DataFrame`: https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#mlflow-pyfunc.

If we need the Keras model to map numpy arrays to numpy arrays, it seems like the custom pyfunc wrapper should load the Keras model using `mlflow.keras.load_model` instead.

Reply:
Yeah that's a good point. However this is how it is currently implemented in mlflow.keras flavor. I guess I should not be changing the name here in this PR anyways.